src/ui/view/dataverse/component/governanceDescription/governanceDescription.scss

```diff
@@ -0,0 +1,44 @@
+@import '@/ui/style/fonts.scss';
+@import '@/ui/style/theme.scss';
+@import '@/ui/style/mixins.scss';
+
+.okp4-dataverse-portal-data-space-governance-description-main {
+  @include with-theme {
+    justify-content: center;
+    padding: 35px 0;
+
+    @include use-breakpoints(('mobile')) {
+      padding-bottom: 20px;
+    }
+
+    .okp4-dataverse-portal-data-space-governance-description-container {
+      display: flex;
+      flex-direction: column;
+      row-gap: 14px;
+      width: 85%;
+
+      @include use-breakpoints(('mobile')) {
+        width: 82%;
+        row-gap: 20px;
+      }
+
+      .okp4-dataverse-portal-data-space-governance-description-title {
+        @include norse-font(700);
+        color: themed('text');
+        font-size: 24px;
+      }
+
+      .okp4-dataverse-portal-data-space-governance-description-description {
+        @include noto-sans(300);
+        color: themed('text');
+        font-size: 16px;
+        line-height: 22px;
+        margin-bottom: 16px;
+
+        @include use-breakpoints(('mobile')) {
```

Comment:
Instead of unsetting CSS values using mobile breakpoint, you can do it using the "mobile first" approach, by starting with mobile style and add style for larger breakpoints.

Reply:
Good point 🙂